src/modules/views/Drive/DriveFolderView.jsx

```diff
@@ -109,22 +109,15 @@ const DriveFolderView = () => {
     sortAttribute: sortOrder.attribute,
     sortOrder: sortOrder.order
   })
-  // Không hiển thị thùng rác trong danh sách file nữa vì đã chuyển sang sidebar
```

Comment:
oh ok removed on this commit, don't hesitate to clean commits or squash them is necessary